src/megatron/bridge/recipes/deepseek/deepseek_v3.py

```diff
@@ -302,3 +304,20 @@ def pretrain_config(
         )
 
     return cfg
+
+
+def pretrain_config_32nodes(**kwargs):    
```

Comment:
is this intended to be used anywhere outside of internal convergence testing? if not, we can expose this config just through `train_from_recipe` to get these overrides instead